k8s/migration/internal/controller/migrationplan_controller.go

```diff
@@ -756,6 +761,25 @@ func (r *MigrationPlanReconciler) TriggerMigration(ctx context.Context,
 		if err != nil {
 			return fmt.Errorf("failed to create Firstboot ConfigMap for VM %s: %w", vm, err)
 		}
+
+		// VDDK files check
+		vddkExpectedDir := "/home/ubuntu/vmware-vix-disklib-distrib"
```

Comment:
Instead of hardcoding to `/home/ubuntu` use the homedir() package from go to find the homedir and append it to the string `vmware-vix-disklib-distrib` 
Also introduce a helper function that you should call to populate this variable.